### DISCUSSION TITLE
Network adapter

### DISCUSSION THREAD

**Discussion by VisaHolder** [filecxx/FileCentipede#554] [Ideas]
Let traffic go through specific selected Adapter.

Ie: I Would like to use my Main WireGuard Adapter instead of proxy. 

All this is missing imo

**Comment by filecxx** [comment]
I'll add this.

**Reply by VisaHolder** [reply]
> I'll add this.

Perfect, Thank you. 

May I ask when you think you will be pushing the next updates? Looking forward to the switch

**Reply by filecxx** [reply]
Few months.
